spec/support/shared_examples_for_api.rb

```diff
@@ -47,6 +47,10 @@
 end
 
 RSpec.shared_examples 'malformed or not iso8601 compliant dates' do |options|
+  subject(:post_to_validate_endpoint) do
+    post options.fetch(:relative_endpoint), valid_params, format: :json
```

Comment:
Is there a reason why `options[:relative_endpoint]` cannot be used here

Reply:
`options.fetch` was being used in other places in this file already. I've changed them all to `options[]` now 🙂